stock_product_pack/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,6 @@
+To configure this module, you need to:
+
+#. Go to *Inventory -> Master Data -> Product*, create a product and check
+   "Is Pack?" field, then set the products has to include in the Pack.
+#. Set type of product pack.
```

Comment:
Wrong order
#. Go to *Inventory -> Master Data -> Product*, create a product and check
   *Is Pack?* field then set *Pack Type* and *Pack component price*.
#. Choose products to include in the pack.